build-frontend/action.yml

```diff
@@ -83,7 +83,7 @@ runs:
         password: ${{ inputs.cloud_password }}
         dockerfile: .vuestorefrontcloud/docker/nextjs/Dockerfile-frontend
         buildoptions: --compress
-        buildargs: NPM_EMAIL,NPM_PASS,NPM_USER,NPM_REGISTRY,NEXT_PUBLIC_API_BASE_URL,NEXT_PUBLIC_MULTISTORE_ENABLED,NEXT_IMAGE_PROVIDER,NEXT_PUBLIC_IMAGE_LOADER_FETCH_URL,NEXT_PUBLIC_IMAGE_LOADER_UPLOAD_URL,NEXT_PUBLIC_COVEO_ORGANIZATION_ID,NEXT_PUBLIC_COVEO_ACCESS_TOKEN,NEXT_DEFAULT_HTML_CACHE_CONTROL,NEXT_PUBLIC_COVEO_TRACKING_ID
+        buildargs: NPM_EMAIL,NPM_PASS,NPM_USER,NPM_REGISTRY,NEXT_PUBLIC_API_BASE_URL,NEXT_PUBLIC_MULTISTORE_ENABLED,NEXT_IMAGE_PROVIDER,NEXT_PUBLIC_IMAGE_LOADER_FETCH_URL,NEXT_PUBLIC_IMAGE_LOADER_UPLOAD_URL,NEXT_PUBLIC_COVEO_ORGANIZATION_ID,NEXT_PUBLIC_COVEO_ACCESS_TOKEN,NEXT_DEFAULT_HTML_CACHE_CONTROL,NEXT_PUBLIC_COVEO_TRACKING_ID,NEXT_PUBLIC_CDN_CACHE_BUSTING_ID
```

Comment:
I believe since you have to pass this variable while using this action in your workflow, you need to add variable also to `inputs` on top of action

Reply:
If I understand you correctly @Szymon-dziewonski you mean, that `NEXT_PUBLIC_CDN_CACHE_BUSTING_ID` should have dedicated input?

If so, that might be a good idea, but as I see in the others @michaelKurowski PRs on this subject it will be prepopulated using current Git commit SHA. So, dedicated input might also non-very often used.

Reply:
The purpose of the cache busting id is to be tied with a version, I feel that it would be too granular to be useful